app/controllers/admin/alert_types/ratings_controller.rb

```diff
@@ -53,14 +54,21 @@ def content_params
           content: [
             :colour,
             :pupil_dashboard_title, :teacher_dashboard_title, :page_title, :page_content,
-            :sms_content, :email_title, :email_content
+            :sms_content, :email_title, :email_content, :chart_type
           ]
         )
       end
 
       def set_template_variables
         @template_variables = @alert_type.cleaned_template_variables
       end
+
+      def set_available_charts
+        available_chart_variables = @alert_type.class_name.constantize::TEMPLATE_VARIABLES.select { |_key, values| values[:units] == :chart }
```

Comment:
This is obvs. a bit yuck at the moment - we get hold of all of the template variables, but as the chart ones reference the variable rather than the chart config we name, we then need to: